server/utils/stripHtml.util.js

```diff
@@ -0,0 +1,11 @@
+import stripHtml from 'string-strip-html';
```

Comment:
I understand fro what you are doing that this strips HTML tags, but will also like to clarify if the library also caters for CSS or Javascript snippets.

Reply:
Yes it caters for both